packages/backend/src/peripherals/database/ZksyncTransactionRepository.ts

```diff
@@ -113,25 +113,20 @@ export class ZksyncTransactionRepository extends BaseRepository {
     return _.zip(noNextBlockNumbers, noPrevBlockNumbers) as [number, number][]
   }
 
-  async getDailyTransactionCount(maxTimestamp: UnixTime) {
+  async refresh() {
```

Comment:
Please rename these `refresh` methods to something more specific e.g. `refreshDailyTransacionCount()` - this applies to both repositories.

Reply:
Done